CMakeLists.txt

```diff
@@ -149,6 +151,7 @@ export(PACKAGE DataFrame)
 # Tests
 if(HMDF_TESTING)
     add_subdirectory(test)
+    file(COPY ${DATA_FILES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/test)
 endif()
```

Comment:
Don't you have to do the same for `HMDF_EXAMPLES`?

Reply:
Yes.. see below.